src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1026,9 +1263,70 @@ private static GencodeFuncotation createIntronFuncotation(final VariantContext v
             );
         }
 
+        // Set our version:
+        gencodeFuncotationBuilder.setVersion(version);
+
         return gencodeFuncotationBuilder.build();
     }
 
+    /**
+     * Get the bases around the given variant (as specified by {@code refAllele} and {@code altAllele}).
+     * The number of bases before and after the variant is specified by {@link #referenceWindow}.
+     * @param refAllele The reference {@link Allele} for the variant.
+     * @param altAllele The alternate {@link Allele} for the variant.
+     * @param reference The {@link ReferenceContext} for the variant, with the current window around the variant.
+     * @param strand The {@link Strand} on which the variant occurs.
+     * @return A {@link String} of bases of length {@link #referenceWindow} * 2 + |variant|
+     */
+    private static String getReferenceBases(final Allele refAllele, final Allele altAllele, final ReferenceContext reference, final Strand strand ) {
+
+        // Adjustment for the start of the variant based on whether it's a deletion or not.
+        // NOTE: This is an artifact of the GATK representation of variants (that they must be at least 1 base)
+        final int deletionStartAdjustment =  GATKVariantContextUtils.isDeletion(refAllele, altAllele) ? 1 : 0;
+
+        if ( strand == Strand.POSITIVE ) {
+
+            final int endWindow =
+                    refAllele.length() >= altAllele.length() ?
+                            referenceWindow + refAllele.length() - 1 :
+                            referenceWindow + altAllele.length() - 1;
+
+            // Get the reference sequence:
+             return new String(reference.getBases(
+//                    new SimpleInterval(reference.getWindow().getContig(),
```

Comment:
Can you get rid of code that has been commented out?

Reply:
Sure.